src/tests/ftest/daos_test/suite.py

```diff
@@ -408,3 +408,18 @@ def test_daos_dedup(self):
         :avocado: tags=daos_test,daos_core_test,test_daos_dedup
         """
         self.run_subtest()
+
+    def test_daos_upgrade(self):
+        """Jira ID: DAOS-1568
```

Comment:
Maybe should be DAOS-10681?

Reply:
ah, sure. though it seems all test cases here are using DAOS-1568.

Reply:
Yeah, probably just copy-paste :)